x-pack/metricbeat/module/azure/storage/integration_tests/storage_integration_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build integration
+// +build azure
+
+package integration_tests
```

Comment:
Could this test be placed in the package it is testing (in ` x-pack/metricbeat/module/azure/storage`)?

Reply:
yes, just a preference I had when building the POC